TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Mappings/UserMapping.cs

```diff
@@ -35,6 +35,8 @@ public class ApplicationUserMapping : IEntityTypeConfiguration<ApplicationUser>
     public void Configure(EntityTypeBuilder<ApplicationUser> builder)
     {
         builder.Property(e => e.ApiRoles).HasColumnType("varchar[]");
+        builder.Property(builder => builder.OneLoginClientId).HasMaxLength(50);
+        builder.Property(builder => builder.OneLoginPrivateKeyPem).HasMaxLength(2000);
```

Comment:
Nit, can we name the param the same as the existing calls?
```suggestion
        builder.Property(e => e.OneLoginClientId).HasMaxLength(50);
        builder.Property(e => e.OneLoginPrivateKeyPem).HasMaxLength(2000);
```